package/src/plugins/add-dts.ts

```diff
@@ -0,0 +1,11 @@
+import { definePlugin } from "../core/define-plugin.js";
+import { addDts } from "../utilities/add-dts.js";
+
+export const addDtsPlugin = definePlugin({
```

Comment:
Not a big fan of calling these `{x}Plugin` like... We know they are plugins, and we import them from `aik/plugins`... I don't know I'm up for a debate on this. But I feel the names could just be `addDts` and import `import { addDts } from 'astro-integration-kit/plugins'`

Reply:
The problem with is that people that want to use standalone utilities will have the wrong one recommended first because of the alphabetical sort

Reply:
@jdtjenkins what do you think?